frontend/src/diagram/sprotty/views/ListView.tsx

```diff
@@ -55,6 +55,7 @@ export class ListView extends RectangularNodeView {
       default:
         break;
     }
+    const radius = nodeStyle.borderRadius || 0;
```

Comment:
how can the field be null when GraphQL says it's required? No need to use ` || 0`